README.md

```diff
@@ -17,234 +17,7 @@ This package provides some CLI tools for working with cplace code.
-                Also merge into the branches of the given customer. The customer name must match the same pattern
-                as mentioned in --all-customers.
-
-            --show-files
-                List files to be merged
-
-        refactor <subcommand> --plugin|-p <plugin>
-            Handles plugin specific refactorings where <subcommand> is one of the following:
-            --test-sources
-                Will refactor an old plugin source structure like 'src/classes' or 'src/java' to a proper Maven-like
-                structure with 'src/main/java' and 'src/test/java'
-
-        e2e [--baseUrl <baseUrl>] [--context <context>] [--tenantId <tenantId>] [--e2eToken <token>]
-            [--browser <browser>] [--plugins <plugins>] [--specs <specs>] [--timeout <timeout>] [--headless]
-            [--noInstall] [--jUnit <?reportsPath>] [--screenshot <?screenshotPath>] [--allure <?allureOutputPath>]
-
-            --baseUrl to configure where to run the tests against (default: 'http://localhost:8083')
-            --context to define in which context cplace is running (default: '/intern/tricia/')
-            --tenantId to define against which tenant the tests are run (default: '' single tenant mode)
-            --e2eToken to define the Test Setup Handler E2E token (default: '')
-            --browser to specify which browser to use (default: Chrome)
-            --plugins to specify a comma separated list of plugins to run tests for
-              (default: all plugins in the current repository)
-            --specs to specify the pattern used to search for specification files inside plugins
-              (default: '**/*.spec.ts')
-            --timeout to specify a global Timeout for Test Execution
-            --headless currently only possible in Chrome and Firefox
-            --noInstall will skip the check for new Selenium Drivers (required to run offline, default: false)
-            --jUnit will create jUnit reports and allows you to specify the location where the reports are stored
-              (default: './e2eJunitReports')
-            --screenshot will create Screenshots on each failed Test and store them in given path
-              (default: './e2eScreenshots')
-            --allure will create Allure Reporter output files and store them in the given path
-              (default: './allure-output')
-            --logLevel will adjust the logLevel in wdio configuraiton
-              (default: 'error', supported options: 'trace | debug | info | warn | error | silent')
-            --specFileRetries The number of times to retry the entire specfile when it fails as a whole
-              (default: 0)
-            --chromeDriverVersion will adjust the version of the Chrome driver
-              (default: latest version)
-
-    Global options:
-        --verbose
-            Print verbose information to console
-```
+The user documentation for the cplace CLI is located [in the Knowledge Base.](/cplace-cli/)
```

Comment:
Since the target is not versioned, you might use a Hugo-specific link here.

Reply:
I see. Since this file is outside of the Knowledge Base, you cannot use a Huo link here.

Reply:
But the relative link was also wrong. I added an absolute link to the KB now (it's not working as the target doesn't exist yet).